brutils/phone.py

```diff
@@ -0,0 +1,37 @@
+# FORMATTING
+############
+
+
+# OPERATIONS
+############
+
+
+def is_valid_landline(phone_number):  # type: (str) -> bool
+    return (
+        isinstance(phone_number, str)
+        and phone_number.isdigit()
+        and len(phone_number) == 10
+        and phone_number[0] != "0"
+        and phone_number[1] != "0"
+        and (
+            phone_number[2] == "2"
+            or phone_number[2] == "3"
+            or phone_number[2] == "4"
+            or phone_number[2] == "5"
```

Comment:
Mini desafio: tentar trocar essas 4 linhas por 1 só, utilizando o operador `in` (membership operator): https://wiingy.com/learn/python/python-membership-and-identity-operators/